lib/hwm/hwm.go

```diff
@@ -120,6 +139,15 @@ func (trie *HighWatermarkTracker) DeleteOldKeys(hwmLimit uint64) []string {
 	return deletedKeys
 }
 
+func (trie *HighWatermarkTracker) ToMap() map[string]uint64 {
```

Comment:
Hm, I thought the Go test philosophy was to include tests in the same package as the thing they tested anyway (though I could be wrong!)